console/console-init/ui/src/modules/address-space/components/AddressSpaceReview/AddressSpaceReview.tsx

```diff
@@ -28,26 +32,84 @@ export interface IAddressSpaceReviewProps {
   data: {
     addressSpaceCommand: string;
   };
+  protocols?: Array<String>;
+  customizeEndpoint: boolean;
+  addRoutes: boolean;
+  tlsCertificate?: string;
 }
 
-const Style = StyleSheet.create({
+const style = StyleSheet.create({
   left_padding: {
     paddingLeft: 32
   },
   bottom_padding: {
     paddingBottom: 16
+  },
+  left_margin_gridItem: {
+    marginBottom: 16,
+    marginLeft: 10,
+    marginRight: 5
+  },
+  preview_info_gridItem: {
+    marginBottom: 16,
+    marginRight: 5
   }
 });
+interface IReviewGridProps {
+  labelText: string;
+  value?: string;
+  valueId: string;
+  addLeftMargin?: boolean;
+}
 
+const ReviewGridItem: React.FunctionComponent<IReviewGridProps> = ({
+  labelText,
+  valueId,
+  value,
+  addLeftMargin = false
+}) => {
+  const className = addLeftMargin
```

Comment:
I would like to suggest use classNames for conditional css.
const cssClassValue= classNames({
    [css(style.preview_info_gridItem)]:addLeftMargin,
    [css(style.left_margin_gridItem)]:!addLeftMargin
});